components/Git/ArticleEditor.tsx

```diff
@@ -123,21 +114,21 @@ export class ArticleEditor extends Component {
       if (meta[1]) this.setPostMeta(meta[1]);
     }
 
-    if (this.core) this.core.raw = content;
+    if (this.editorContent) this.editorContent = content;
```

Comment:
```suggestion
    this.editorContent = content;
```
以前需要判空是因为 `this.core` 是用来存组件实例对象的，只有存在才能存取它的属性。

而现在你是直接存取编辑器里的数据，就直接存取就好了。

Reply:
还有不止一处同类代码